tifeatures/settings.py

```diff
@@ -21,6 +21,7 @@ class _APISettings(pydantic.BaseSettings):
     """API settings"""
 
     name: str = "TiFeatures"
+    DEBUG: bool = False
```

Comment:
👀 ?

Reply:
It makes it much easier to debug what's getting into the catalog